feature-libs/order/components/order-details/order-detail-billing/order-detail-billing.component.html

```diff
@@ -1,11 +1,15 @@
 <div class="cx-order-items container" *ngIf="order$ | async as order">
-  <ng-template
-    *ngIf="order.paymentInfo"
-    [cxOutlet]="cartOutlets.PAYMENT_INFO"
-    [cxOutletContext]="{
-      readonly: true,
-      paymentDetails: order.paymentInfo
-    }"
-  >
-  </ng-template>
+  <div class="cx-review-summary" *ngIf="order.paymentInfo as paymentDetails">
+    <div class="cx-review-summary-card cx-review-summary-payment-card">
```

Comment:
i'm not sure if styling like cx-review-summary-card is accessible here. we can check when sampledata is released